test/whereEq.js

```diff
@@ -45,7 +45,6 @@ describe('whereEq', function() {
 
   it('is true for an empty spec', function() {
     eq(R.whereEq({}, {a: 1}), true);
-    eq(R.whereEq(null, {a: 1}), true);
```

Comment:
Yes, I'd say so. I removed this assertion because it was failing, and we shouldn't support `null` here.